documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="collected-logs-cr-info_{context}"]
+= Collected logs and custom resource information
+
+You can download logs and custom resource (CR) `yaml` files for the following targets by using the {project-short} web console or command line interface (CLI):
+
+* Migration plan: Web console or CLI.
+* Virtual machine: Web console or CLI.
+* Namespace: CLI only.
+
+The `must-gather` tool collects the following logs and CR files in an archive file:
+
+* Logs:
+** `importer` pod: Data volume conversion log.
```

Comment:
@istein1 Is this OK as a description of the importer pod?

Reply:
Importer pod; Disk transfer into data volume log

Reply:
That is what I meant. But I can make it clearer.